README.md

```diff
@@ -1,216 +1,265 @@
-- run `./cleanup.sh` in your terminal
-- change the setting
-- open the project in Android Studio again
+</details>
 
-For configuring advanced settings, you can use Xcode. Open the `iosApp/iosApp.xcworkspace` in Xcode after opening the project in Android Studio, and use Xcode to make your changes.
+To run the application, set the `TEAM_ID`:
+
+1. In the template, navigate to the `iosApp/Configuration/Config.xcconfig` file.
+2. Set your `TEAM_ID`.
+3. Re-open the project in Android Studio. It should show the registered iOS device in the `iosApp` run configuration.
+
+## Make your first changes
+
+1. In Android Studio, navigate to the `shared/src/commonMain/kotlin/App.kt` file.
+   It's the common entry point for your Compose Multiplatform app. 
+   
+   Here, you see the code responsible for rendering the "Hello, World!" button and the animated Compose Multiplatform logo:
+
+   ```kotlin
+   @OptIn(ExperimentalResourceApi::class)
+   @Composable
+   internal fun App() {
+       MaterialTheme {
+           var greetingText by remember { mutableStateOf("Hello, World!") }
+           var showImage by remember { mutableStateOf(false) }
+           Column(Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
+               Button(onClick = {
+                   greetingText = "Hello, ${getPlatformName()}"
+                   showImage = !showImage
+               }) {
+                   Text(greetingText)
+               }
+               AnimatedVisibility(showImage) {
+                   Image(
+                       painterResource("compose-multiplatform.xml"),
+                       null
+                   )
+               }
+           }
+       }
+   }
+   ```
+
+2. Update the shared code by adding a text field that will update the name displayed on the button:
+
+   ```diff
+   @OptIn(ExperimentalResourceApi::class)
+   @Composable
+   internal fun App() {
+       MaterialTheme {
+           var greetingText by remember { mutableStateOf("Hello, World!") }
+           var showImage by remember { mutableStateOf(false) }
+           Column(Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
+               Button(onClick = {
+                   greetingText = "Hello, ${getPlatformName()}"
+                   showImage = !showImage
+               }) {
+                   Text(greetingText)
+               }
+   +           TextField(greetingText, onValueChange = { greetingText = it })
+               AnimatedVisibility(showImage) {
+                   Image(
+                       painterResource("compose-multiplatform.xml"),
+                       null
+                   )
+               }
+           }
+       }
+   }
+   ```
+
+3. Re-run both `androidApp` and `iosApp` configurations. You'll see this change reflected on both the Android and iOS apps:
+
+   <img src="readme_images/text_field_added.png" height="200px">
+
+## How to configure the iOS application
```

Comment:
Why do we need this section in Getting started? IMO it dedicates a separate doc article.

Reply:
Well since we don't have docs, I guess the only option is remove it

Reply:
This is actually part of how the specific template is set up, and does not _necessarily_ apply to a general Compose Multiplatform project.

Reply:
I tried a new phrasing so that it's clear that it's not a part of the tutorial, but an explanation of the setup

Reply:
LGTM.

Reply:
LGTM